pkg/api/v1/handlers/pagination.go

```diff
@@ -12,15 +12,22 @@ const (
 )
 
 // getPaginationOptions returns a ListOptions struct with validated pagination parameters
-func getPaginationOptions(page int) *models.ListOptions {
+func getPaginationOptions(page int, includeDeleted ...bool) *models.ListOptions {
```

Comment:
why is this a slice of `bool`?  if this is to just make `includeDeleted` an optional argument I think we should handle it differently. 

Let's make a follow up issue to discuss this.

Reply:
I agree with that and it must not be here in the paginationOptions. I personally think we should get rid of that includeDeleted at all. because we do not actually delete any resources instances and tasks are terminated or done. but not deleted. and about the projects we can discuss it.

Reply:
fixed